arch/arm/boot/dts/overlays/README

```diff
@@ -3823,6 +3823,26 @@ Params: int_pin                 GPIO used for INT (default 25)
         cs                      SPI bus Chip Select (default 0)
 
 
+Name:   waveshare-can-fd-hat-mode-a
+Info:   Overlay for the Waveshare 2-Channel Isolated CAN FD Expansion HAT
+        for Raspberry Pi, Multi Protections. Use this overlay when the
+        HAT is configured in Mode A (Default), with can0 on spi0.0 
```

Comment:
There's a stray space at the end of this line...

Reply:
corrected